### PR TITLE
30.0.0

### PR DESCRIPTION
**Pull request by Altahrim**

### CODE REVIEW

File: config/config.php [hunk 1]
```diff
@@ -1,6 +1,38 @@
...
+				'signature' => 'QjU8BAz/b+ZSv1CFaom3VHE5IbT8giQgJ6/N00jPnWlRD8daXIsmIoLRe9UMl1yi
+hTRRW3r7MQCa6RklmtZIAxsoRYlk7eEdbogKTODLRe9gsXdXF2KFisADl6bcEw1H
+MLyVf7ncFnFlp9GYwpUEcGvJAzIcckkGTmBHAMTfmpSNQoijzdCP9ytKP2tJSe7h
+tOXDKB0sOYzsO5A725ERye0RKfgMY7TRG0vq9tIxB7ydSykxVAGglKHDwl/6mn5D
+NUfRJKep0nzpJXIOz+BiE6tTRxYMeRkUsI2WLPnPC3Ltu2Xl0u3iOoWaDfdD0Ox9
+XjOoUiZRQ+fUfAGTnEZ+Lg==',
+			],
+		],
+		'29.0.7.1' => [
+			'100' => [
```
Comment (nickvergessen) [inline config/config.php:21]:
```suggestion
			'30' => [
```

Comment (Altahrim) [reply config/config.php:21]: I always forget this one...

File: config/config.php [hunk 2]
```diff
@@ -644,38 +676,6 @@
...
-				'signature' => 'sCkQx6CgJezLvF87Nkdtx83qLvW6Lnj+U6Lz0lKZocn1E957y/pnkUmWXs4FU33n
-4tW9b6PGQJTCkP0jWqN7tEc80VII2ZrCESOzI3F8uQDYqoVWeOLxP7gYorXEh+oY
-Fp5NU9eIsfipcH0AMGtBtuFy1+1mYeSfSMoFl5E/7CPaOQksD9MS4NRtQ6TYfK7L
-W0yoo+NGcf9WGQ75z4g1p7l7+5tvpQ2a794Bg0eIIZePNUrJ/puqJ53x7nQ2d41U
-qjIENY7lso6I1X+ggz2tFegXab7cRP0uUIelgOJMV9rPZIGfqMwmj5gWlcr1a847
-/V2ry0elGhEPZAqn9zzBuw==',
-			],
-		],
-		'29.0.7.1' => [
-			'100' => [
```
Comment (nickvergessen) [inline config/config.php:664]: need to keep this with 100, so all beta users get the final?

Comment (Altahrim) [reply config/config.php:664]: Yes, fixed the issue in config builder!

**Review by AndyScherzinger — APPROVED** [review]

**Pull request merged**